### DISCUSSION TITLE
Callbacks issue

### DISCUSSION THREAD

**Discussion by SWeiCHEN** [apache/airflow#44594] [General]
### What do you see as an issue?

There's are some new parameter for DAG callback setting. E.g. `on_skipped_callback` added since version 2.9, but still can not use this parameter and not found in this repo.
https://airflow.apache.org/docs/apache-airflow/2.10.2/administration-and-deployment/logging-monitoring/callbacks.html

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.

**Comment by potiuk** [comment]
You have not explained what problem you had (and whether it's an airflow issue it's unnknown) when you tried, so I am converting it into discussion.

THis is the description of the callback - you likely did not read that documentation (but this is a wild guess). If that's the case and you expected the callback to be called in a different place, we will appreciate if you click "suggest a change on this page" - on the page where you missed the callback listing and update it and make a PR fixing it.

```
    :param on_skipped_callback: much like the ``on_failure_callback`` except
        that it is executed when skipped occur; this callback will be called only if AirflowSkipException get raised.
        Explicitly it is NOT called if a task is not started to be executed because of a preceding branching
        decision in the DAG or a trigger rule which causes execution to skip so that the task execution
        is never scheduled.
```

**Reply by SWeiCHEN** [reply]
is `on_skipped_callback` can only set under task or can be DAG setting also?

**Reply by potiuk** [reply]
Looking at teh BaseOperator docstring (I heartily invite to do so) - it looks it's defined there. If you have any problem with that, then the ball is in your court to describe behaviour you see and do not agree with. With details, logs, examples what you've done and how you tried to debug it. 

It's virtually '0' chance someone will help (in their free time) to answer your problems if you state 'I have a problem'. But if you spend your time on describing the problem and showing that you've done your part, there is a chance, someone - at this free forum, for free software that you paid 0 for, might spend their free time away of their family and day job to actually help you.

But you have to do your part to describe your problem first if you want to suceed in convincing someone to help you.

**Reply by potiuk** [reply]
BTW. Generally speakin (as all other cases in airflow, you can use `default_args` if you want to define something that applies to all tasks. Maybe that is what you missed? But I am wildly guessing as you have not described your problem.
